confidant/routes/jwks.py

```diff
@@ -6,11 +6,14 @@
 from confidant.services.jwkmanager import jwk_manager
 from confidant.schema.jwks import jwt_response_schema, JWTResponse, \
     jwks_list_response_schema, JWKSListResponse
-
+from confidant.settings import JWT_MAPPING_MODULE
+from confidant.utils import misc
 
 logger = logging.getLogger(__name__)
 blueprint = blueprints.Blueprint('jwks', __name__)
 
+jwt_mapping_check = misc.load_module(JWT_MAPPING_MODULE)
+
 
 @blueprint.route('/v1/jwks/token', methods=['GET'])
 @authnz.require_auth
```

Comment:
we need to change this to

```python
def get_token(id):
```

the `id` will the be the token (resource_id) we're attempting to request.

Reply:
so this id will be the envoy_service name?

Reply:
@Yueren-Wang yes